.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # Default
 
-* @evan-gray
+- @evan-gray
```

Comment:
pretty sure this has to stay a star. 🙃 maybe prettier thought this was markdown

Reply:
Removed CODEOWNERS from this PR. Will do it in a separate one.